synapse/config/logger.py

```diff
@@ -55,34 +55,43 @@
         format: '%(asctime)s - %(name)s - %(lineno)d - %(levelname)s - \
```

Comment:
btw: remember that the docker image has its own default log config; please update it to keep it in sync.